tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -653,18 +688,19 @@ public Counter<T> getTotals(EnumSet<Organization> conflictedOrganizations) {
                 if (weight == 0) {
                     continue;
                 }
+                annotateTranscript("-- Considering %s which has %d item(s)", entry.getKey().getDisplayName(), items.size());
```

Comment:
suggestion (if you're not already planning on it): don't hard-code the initial hyphens or their count, instead each VoteResolverTranscript should track its indentation level, and the conversion from 2 into "-- " should be in its own method

Reply:
really the conversion from 2 into "-- " should happen on the front end -- the back end should provide structured json with separate count and message

Reply:
This is somewhat of a tech preview. I'd like to improve how it's explained beyond just a prose text blob. So this is a start so that we can see that the right things are being explained. That's a good refactoring but let's wait to see how this shapes up.